Vagrantfile

```diff
@@ -46,7 +53,7 @@ Vagrant.configure(2) do |config|
   config.vm.hostname = options['machine_name']
 
   # network settings
-  config.vm.network 'private_network', ip: options['ip']
+  config.vm.network 'private_network', ip: options['netmask']+'.'+options['ip']
```

Comment:
It is also not a subnet but a prefix, see https://github.com/yiisoft/yii2-app-advanced/pull/223#discussion_r96139333